orchestrator-salt/src/main/resources/salt/salt/nginx/restoressl.sls

```diff
@@ -0,0 +1,18 @@
+restore_last_ssl_conf:
+  module.run:
+    - name: file.restore_backup
+    - path: /etc/nginx/sites-enabled/ssl.conf
+    - backup_id: 0
+
+touch_ssl_conf_for_restart:
+  file.touch:
+    --name: /etc/nginx/sites-enabled/ssl.conf
```

Comment:
Sanity check: Is the `--name` possibly a typo? Was it rather meant as a list element `- name`?

Reply:
yes, fixed